setup.py

```diff
@@ -23,13 +23,10 @@
     classifiers=[
         'Intended Audience :: Developers',
         'Operating System :: OS Independent',
-        'Programming Language :: Python :: 2',
-        'Programming Language :: Python :: 2.7',
-        'Programming Language :: Python :: 3',
```

Comment:
You could have kept this but of not much consequence 👌

Reply:
Oh yeah thanks! I'll put it back 😊